index.html

```diff
@@ -1065,71 +1127,73 @@ <h1 class="title is-size-4">Components</h1>
           </ul>
         </div>
         <div class="column is-3">
-          <div class="is-3 title"><b>News</b></div>
+          <h3 class="is-3 title"><b>News</b></h3>
           <ul>
-            <li><a>Labore et dolore magna aliqua</a></li>
-            <li><a>Kanban airis sum eschelor</a></li>
-            <li><a>Modular modern free</a></li>
-            <li><a>The king of clubs</a></li>
-            <li><a>The Discovery Dissipation</a></li>
-            <li><a>Course Correction</a></li>
-            <li><a>Better Angels</a></li>
+            <li><a tabindex="0">Labore et dolore magna aliqua</a></li>
+            <li><a tabindex="0">Kanban airis sum eschelor</a></li>
+            <li><a tabindex="0">Modular modern free</a></li>
+            <li><a tabindex="0">The king of clubs</a></li>
+            <li><a tabindex="0">The Discovery Dissipation</a></li>
+            <li><a tabindex="0">Course Correction</a></li>
+            <li><a tabindex="0">Better Angels</a></li>
           </ul>
         </div>
         <div class="column is-4">
-          <div class="is-3 title"><b>Help</b></div>
+          <h3 class="is-3 title"><b>Help</b></h3>
           <ul>
-            <li><a>Objects in space</a></li>
-            <li><a>Playing cards with coyote</a></li>
-            <li><a>Goodbye Yellow Brick Road</a></li>
-            <li><a>The Garden of Forking Paths</a></li>
-            <li><a>Future Shock</a></li>
+            <li><a tabindex="0">Objects in space</a></li>
+            <li><a tabindex="0">Playing cards with coyote</a></li>
+            <li><a tabindex="0">Goodbye Yellow Brick Road</a></li>
+            <li><a tabindex="0">The Garden of Forking Paths</a></li>
+            <li><a tabindex="0">Future Shock</a></li>
           </ul>
         </div>
       </div>
 
       <div class="content has-text-centered">
         <div class="soc">
-          <a><i class="fab fa-github-alt fa-2x"></i></a>
-          <a><i class="fab fa-youtube fa-2x"></i></a>
-          <a><i class="fab fa-facebook fa-2x"></i></a>
-          <a><i class="fab fa-twitter fa-2x"></i></a>
+          <a tabindex="0"><i class="fab fa-github-alt fa-2x"></i> <div class="visually-hidden">github</div></a>
+          <a tabindex="0"><i class="fab fa-youtube fa-2x"></i> <div class="visually-hidden">youtube</div></a>
+          <a tabindex="0"><i class="fab fa-facebook fa-2x"></i> <div class="visually-hidden">facebook</div></a>
+          <a tabindex="0"><i class="fab fa-twitter fa-2x"></i> <div class="visually-hidden">twitter</div></a>
         </div>
-        <p>The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.</p>
+        <p>The source code is licensed <a tabindex="0" class="third-party-resources" tabindex="0" href="http://opensource.org/licenses/mit-license.php" target="_blank"><i class="fas fa-external-link-alt"></i> MIT <div class="visually-hidden">will open in a new tab</div></a>.</p>
```

Comment:
The "will open in a new tab" text should probably be inside the link itself. E.g. <a>MIT <span class="visually-hidden">will open in a new tab</a>

Reply:
Resoled, thanks